autoload/codefmt/zprint.vim

```diff
@@ -26,6 +26,30 @@
 
 let s:plugin = maktaba#plugin#Get('codefmt')
 
+""
+" @private
+function! s:ApplyToZprintOptions(opts, key, value) abort
```

Comment:
Behavior here is a little hard to follow. It could use a doc comment and example input/output.

Reply:
I'll plan to fix this in a separate PR